README.md

```diff
@@ -155,7 +160,7 @@ The tricks to remember are:
 As an example, if you wanted to run the CS checks under PHP 7.4 using locked dependencies, you could do something like the following:
 
 ```bash
-$ docker run -v $(realpath .):/github/workspace -w=/github/workspace laminas-check-runner:latest '{"php":"7.4","deps":"locked","extensions":[],"ini":["memory_limit=-1"],"command":"./vendor/bin/phpcs"}'
+$ docker run -v $(realpath .):/github/workspace -w=/github/workspace ghcr.io/laminas/laminas-continuous-integration:1  '{"php":"7.4","deps":"locked","extensions":[],"ini":["memory_limit=-1"],"command":"./vendor/bin/phpcs"}'
```

Comment:
I am not sure but I guess this was meant to be used locally.
Imho, the `docker build . -t laminas-check-runnter:latest` command is missing somewhere. Then, this line is absolutely okayish.
But I am not sure about this. Maybe @weierophinney can say something about it.

Reply:
I originally wrote this based on having built locally, but I've since switched to using images I've pulled off of ghcr.io. That feels more natural as a docker user (pull and use prebuilt images), so I'm good with this change.